tests/aws/services/kms/test_kms.py

```diff
@@ -435,16 +468,30 @@ def test_sign_verify(self, kms_create_key, snapshot, key_spec, sign_algo, aws_cl
         verification = aws_client.kms.verify(
             MessageType="RAW", Signature=signature["Signature"], Message=plaintext, **kwargs
         )
+
         snapshot.match("verification", verification)
         assert verification["SignatureValid"]
 
+        # Ensure signatures can be verified using the public key and specific params
+        response = aws_client.kms.get_public_key(KeyId=key_id)
+        public_key_bytes = response.get("PublicKey")
+        key = load_der_public_key(public_key_bytes)
+
+        raw_vargs = enforce_salt_length(sign_algo, "RAW")
+
+        key.verify(signature=signature["Signature"], data=plaintext, **raw_vargs)
+
```

Comment:
need to add snapshot validation for these changes.

Reply:
In fact I think the author of this PR is right in testing this way. Testing by doing the verification offline using the Public Key will clearly show why the current behaviour of LocalStack is NOT actually compatible with AWS, even though it claims so. RSAPSS signing in LocalStack is 100% not compatible with AWS KMS.

Reply:
We can rather use the existing code as mentioned in the above comments.